common/src/main/res/values/styles.xml

```diff
@@ -324,6 +324,18 @@
         <item name="android:textSize">14sp</item>
     </style>
 
+    <style name="InterRegular">
+        <item name="fontFamily">@font/inter_regular</item>
+    </style>
+
+    <style name="InterRegular.MG">
+        <item name="android:textColor">@color/light_gray_black</item>
+    </style>
+
+    <style name="InterRegular.MG.32">
+        <item name="android:textSize">32sp</item>
+    </style>
```

Comment:
Should we be using a different naming system for Styles that doesn't use the name of the font?

Reply:
Also, I am wondering if the requirement of a different font is a deliberate choice by the designer.

Reply:
This is correct, we should not be using styles with the font name in them. That's the whole point of the new typography styles - so that we can simply switch the font later and not have to do any other changes.

@ClaudeHangui I think you can continue using Montserrat for your story - we have a different one specifically for changing fonts throughout the entire app.